etl/steps/data/garden/oecd/2023-05-19/co2_air_transport.meta.yml

```diff
@@ -1,12 +1,15 @@
 dataset:
   title: Air Transport CO2 Emissions
-  description: >
-    This database includes annual, quarterly, and monthly information on CO2 emissions related to commercial passenger, freight, and general aviation flights, on both a territory and a residence basis, for 186 countries. These CO2 emissions are estimated by the OECD, based on a consistent methodology across countries.
+  description: This database includes annual, quarterly, and monthly information on
+    CO2 emissions related to commercial passenger, freight, and general aviation flights,
+    on both a territory and a residence basis, for 186 countries. These CO2 emissions
+    are estimated by the OECD, based on a consistent methodology across countries.
   licenses:
   - name: Creative Commons BY 4.0
     url: https://www.oecd.org/termsandconditions/#:~:text=You%20must%20give%20appropriate%20credit,(accessed%20on%20(date)).
   sources:
-  - name: OECD (2023), Air Transport CO2 Emissions
+    name: Clarke et al. (2022)
```

Comment:
I think you need the '-' back in here, I believe that's why the CI tests are failing atm